github/apps_manifest.go

```diff
@@ -41,7 +37,7 @@ func (s *AppsService) CompleteAppManifest(ctx context.Context, code string) (*Ap
 	if err != nil {
 		return nil, nil, err
 	}
-	req.Header.Set("Accept", mediaTypeAppManifestPreview)
+	req.Header.Set("Accept", mediaTypeV3)
```

Comment:
Please remove this line, as this header is added to all requests here:
https://github.com/google/go-github/blob/master/github/github.go#L383